lib/smart_core/types/variadic.rb

```diff
@@ -2,6 +2,8 @@
 
 # @api public
 # @since 0.3.0
+# @version 0.4.0
```

Comment:
this comment is addressed to Variadic constant only. but Variadic constant has no changes in current PR. so, this comment is unnecessary.

Reply:
ur right, done